migrations/20250423114619-populate-overwritten-datasets-size.js

```diff
@@ -0,0 +1,156 @@
+module.exports = {
+  async up(db, client) {
+    await db
+      .collection("Dataset")
+      .find({ size: 0 })
+      .forEach(async (dataset) => {
+        const origDatablocks = await db
+          .collection("OrigDatablock")
+          .find({
+            datasetId: dataset._id,
+          })
+          .toArray();
+        const origDatablockSizeSum = origDatablocks.reduce(
+          (acc, origDatablock) => acc + origDatablock.size,
+          0,
+        );
+
+        const datablocks = await db
+          .collection("Datablock")
+          .find({
+            datasetId: dataset._id,
+          })
+          .toArray();
+
+        const datablockSizeSum = datablocks.reduce(
+          (acc, datablock) => acc + datablock.size,
+          0,
+        );
+
+        const datasetSizeSum = origDatablockSizeSum + datablockSizeSum;
```

Comment:
Are you sure that this is the way that it is calculated?
I think that the dataset total size is the sum of the size of the files listed in the `origDatablock`

Reply:
OK. You are correct.
Let's resolve this conversation, and bring the discussion to the collaborators